snapcraft/plugins/python.py

```diff
@@ -277,17 +287,44 @@ def __init__(self, *, exec_func, runnable, package_dir, env,
         if dependency_links:
             self._extra_pip_args.append('--process-dependency-links')
 
+    def list(self, exec_func=None):
```

Comment:
It replicates the `pip list` command, just like `install`, `download` and `wheel`. It would be inconsistent to change